variables.tf

```diff
@@ -150,3 +150,9 @@ variable "create_health_check_firewall" {
   default     = true
   type        = bool
 }
+
+variable "enable_firewall_logging" {
```

Comment:
Let's use `firewall_enable_logging` for parity with https://github.com/search?p=7&q=org%3Aterraform-google-modules+firewall+logging&type=Code